imxrt-hal/Cargo.toml

```diff
@@ -9,12 +9,12 @@ keywords = ["imxrt", "nxp", "embedded", "no_std"]
 categories = ["embedded", "no-std"]
 license = "MIT/Apache-2.0"
 edition = "2018"
-version = "0.2.0"
+version = "0.2.1"
 
 
 [dependencies]
 cortex-m = { version = "0.6" }
-imxrt-ral = { version = "0.2.0", path = "../imxrt-ral" }
```

Comment:
We shouldn't need to remove the path, but we do need to change the version

Reply:
See https://github.com/tokio-rs/tokio/blob/master/tokio-tls/Cargo.toml for example